integration/direct_submission.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: page
+title: Submitting Calibrations / Direct Submissions
+subtitle: How to submit a Direct submission via the API
+---
+
+# DIRECT submissions
+These special requests can be submitted into the Observation Portal to represent an observation that must take place at a fixed time and location, separate from the scheduler's scheduled observations. For these requests, the `observation_type` is `DIRECT`, denoting that they are a direct submission and excluding them from scheduling runs.
+
+## When to use
+Direct submissions can be used to schedule calibration frames that happen before or after the scheduleable night bounds (daytime calibrations), or to schedule during a night during a period of downtime when the scheduler is not scheduling. It is possible, but not recommended to mix DIRECT submissions during a night with scheduler scheduled observations - for night time calibrations it will work best to schedule them alongside all the science requests. 
```

Comment:
I had to read this a couple of times to totally get it - I might re-write it as:

It is possible, but not recommended to mix DIRECT submissions during a night with scheduled observations (those submitted by an automated scheduler). For night-time calibrations it will work best to utilize the automated scheduler in order to schedule them alongside the science requests, rather than submitting them directly.